test/integration-test/scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/framework/helper/ScalingCaseHelper.java

```diff
@@ -53,19 +53,21 @@ public static long generateSnowflakeKey() {
      * Generate MySQL insert data, contains full fields.
      *
      * @param orderIdGenerate order id generate algorithm
+     * @param orderItemIdGenerate order item id generate algorithm
      * @param databaseType database type
      * @param insertRows insert rows
      * @return insert data list
      */
-    public static Pair<List<Object[]>, List<Object[]>> generateFullInsertData(final AutoIncrementKeyGenerateAlgorithm orderIdGenerate, final DatabaseType databaseType, final int insertRows) {
+    public static Pair<List<Object[]>, List<Object[]>> generateFullInsertData(final AutoIncrementKeyGenerateAlgorithm orderIdGenerate, final AutoIncrementKeyGenerateAlgorithm orderItemIdGenerate, 
+                                                                              final DatabaseType databaseType, final int insertRows) {
```

Comment:
Could we keep one generateAlgorithm parameter, or just remove generateAlgorithm parameter?